src/api/api.test.tsx

```diff
@@ -10,7 +10,7 @@ describe('getItems', () => {
       json: () => Promise.resolve({ docs: itemsData, pages: 1 }),
     } as Response);
 
-    const data = await getItems('');
+    const data = await getItems({ search: '', sorting: SORTING.nameAsc, page: 1, limit: 10 });
```

Comment:
There is a potentially flacky test because you relay on some external api. You can't predict what it sends to you in the future so your test might fail at some time